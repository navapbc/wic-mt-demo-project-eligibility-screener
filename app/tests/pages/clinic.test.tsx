// test/pages/index.test.js
import { render, screen, waitFor } from '@testing-library/react'
import { axe } from 'jest-axe'

import Clinic from '@pages/clinic'

describe('Clinic', () => {
  it('should render the heading', () => {
    render(<Clinic previousRoute="/review" />)

    const heading = screen.getByText(/Choose a clinic/i)

    expect(heading).toBeInTheDocument()
    expect(heading).toMatchSnapshot()
  })

  it('should pass accessibility scan', async () => {
    const { container } = render(<Clinic previousRoute="/review" />)
    const results = await axe(container)

    await waitFor(() => {
      expect(results).toHaveNoViolations()
    })
  })
})
