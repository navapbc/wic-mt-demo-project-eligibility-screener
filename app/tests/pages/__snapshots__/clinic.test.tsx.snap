// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clinic should render the heading 1`] = `
<h2>
  Choose a clinic
</h2>
`;
